install/manifests/scheduledworkflow-controller.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
```

Comment:
What is this for? Why don't you just deploy it with ksonnet?

Reply:
I added to check executions, like [mpi-operator](https://github.com/kubeflow/mpi-operator/tree/master/deploy). And also I think that the components to deploy with ksonnet will be prepared on kubeflow repo.

Reply:
So the path taken by other components like tf-operator is to create ksonnet components in kubeflow/kubeflow and then use them to deploy the operator. 
You can then check in a test app like
https://github.com/kubeflow/tf-operator/tree/master/test/test-app

To deploy the operator as part of tests. 
The downside of this approach 

If you want to check in raw YAML I think that's fine but it means maintaining two sets of configs. You'll need ksonnet anyway for E2E testing.

Reply:
We do want support for E2E testing so let's follow jlewi@ approach to avoid having to maintain two sets of configs.

Reply:
So I will delete this manifest, and try to enable to install pipelines as the component of kubeflow, okay?